pkg/controller/machineset.go

```diff
@@ -427,7 +427,9 @@ func (c *controller) manageReplicas(ctx context.Context, allMachines []*v1alpha1
 		}
 		klog.V(2).Infof("Too many replicas for %v %s/%s, need %d, deleting %d", machineSet.Kind, machineSet.Namespace, machineSet.Name, (machineSet.Spec.Replicas), diff)
 
+		logMachinesWithPriorityAnnotation(activeMachines)
```

Comment:
Actually, each machine will have a priority annotation set. Should we just log machines with priority set to `1` instead?

Reply:
Hmm..ok, I thought it might be useful to see if there was something strange/diff about priority. But yes, to avoid overlog we can limit this to p1.